config.js

```diff
@@ -0,0 +1,39 @@
+
+require('dotenv').config();
+// In this file you can configure migrate-mongo
+let dbName, dbUrl = '';
+if (process.env.NODE_ENV === 'test') {
+  dbName = 'cfh_test';
+  dbUrl = process.env.MONGOHQ_URL_TEST;
+} else if (process.env.NODE_ENV === 'development') {
+  dbName = 'cfh';
+  dbUrl = process.env.MONGOHQ_URL;
+} else if (process.env.NODE_ENV === 'staging') {
+  dbName = 'gondor-cfh-staging';
+  dbUrl = process.env.MONGOHQ_URL_STAGING;
+} else {
+  dbName = 'gondor-cfh';
+  dbUrl = process.env.MONGOHQ_URL_PROD;
+}
+
+module.exports = {
+
+  mongodb: {
+
+    // TODO Change (or review) the url to your MongoDB:
+    url: dbUrl,
+
+    // TODO Change this to your database name:
```

Comment:
Is there any reason we left the TODO comments?

Reply:
I left it there so it'll be easy for others to know how to make changes if necessary. However, looking at it again, the names are quite self explanatory so I'll take those comments out. Thanks for pointing that out.